src/components/Sheet/README.md

```diff
@@ -0,0 +1,203 @@
+---
+name: Sheet
+category: Overlays
+platforms:
+  - web
+keywords:
+  - sheet
+---
+
+# Sheet
+
+A sheet is a large container that enters from the edge of the screen when triggered by the merchant. It’s used to provide merchants with actions and information contextual to the page. It doesn’t interrupt their flow like a modal.
+
+---
+
+## Required components
+
+The sheet component must be wrapped in the [frame](/components/structure/frame) component.
+
+---
+
+## Use in an embedded application
+
+Use of the sheet component in an embedded application is not currently supported. If this is a feature you would like to see supported by Shopify App Bridge, let us know in the [forums](https://ecommerce.shopify.com/c/shopify-apis-and-technology).
+
+---
+
+## Accessibility
+
+Sheets provide an opportunity to let merchants dig into more detail on their current task, or access information for their current task in a different way. Although merchants may be able to see content in the sheet and the main page content at the same time, they should only be expected to interact with one or the other at any given time.
+
+### Keyboard support
+
+- Use the `onClose` prop so that the sheet can be closed with the <kbd>esc</kbd> key as well as with button-based controls
+- Use a button to open the sheet
+- When the sheet opens, move focus to it so merchants who rely on the keyboard and screen readers can access it
+- When the sheet closes, move focus back to the button that launched it
+
+---
+
+## Best practices
+
+The sheet component should:
+
+- Include a heading that summarizes the actions and information in the sheet, for example, More filters
+- Be openable through clear actions, like a link or button
+- Be close-able through clear actions, like Done, the [X] button, and the esc key
+- Include information and actions contextual to the current task
+- Not block merchants from completing their task, like a modal would
+- Not open from within another sheet (only one sheet can be open at a time)
+- Preserve its state—the settings and actions won’t reset when it’s closed
```

Comment:
This is an em dash (hard to see in a monospace font), which we use for interjections. It will look like this in a non-monospace font:

> Preserve its state—the settings and actions won’t reset when it’s closed

I believe this correctly follows Polaris content guidelines.